NetworkingLab/app/src/main/java/com/justinwells/networkinglab/MainActivity.java

```diff
@@ -0,0 +1,137 @@
+package com.justinwells.networkinglab;
+
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+import android.os.AsyncTask;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.Toast;
+
+import com.google.gson.Gson;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+public class MainActivity extends AppCompatActivity {
+
+
+
+    public static final String CEREAL_URL
+            = "http://api.walmartlabs.com/v1/search?query=cereal&format=json&apiKey=mp9c3vfyp8z9z9up6bc746ny\n";
+    public static final String CHOCOLATE_URL
+            = "http://api.walmartlabs.com/v1/search?query=chocolate&format=json&apiKey=mp9c3vfyp8z9z9up6bc746ny";
+    public static final String TEA_URL
+            = "http://api.walmartlabs.com/v1/search?query=tea&format=json&apiKey=mp9c3vfyp8z9z9up6bc746ny\n";
+
+    RecyclerView recyclerView;
+
+
+
+    Button cerealButton, chocolateButton, teaButton;
```

Comment:
All your member variables (and your constants) can be made `private`. It's a good idea to use the most restrictive access you can, in order to avoid unexpected modifications to any of your variables.